cmd/ursrv/static/index.html

```diff
@@ -211,47 +216,48 @@
+      };
+      
+      var baseLayer = L.tileLayer(
+        'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',{
```

Comment:
https ?

Reply:
For second proposition can I have a json file for my test ???